Gemfile

```diff
@@ -11,6 +11,7 @@ gem 'aws-sdk-route53', '~> 1.21.0'
 gem 'aws-sdk-s3', '~> 1'
 gem 'devise', '~> 4.6.2'
 gem 'devise_invitable', '~> 2.0.0'
+gem 'geocoder', '~> 1.5', '>= 1.5.1'
```

Comment:
This dependency isn't used yet, Can you introduce the dependency when you come to implement the code please?